content/docs/manual/overview.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: User Manual
+title: Product Manual
```

Comment:
This still reads - User Manual both on the sidebar and the page on the deploy preview.

Reply:
ok. i'll check what the issue is.